src/web/src/pages/index.tsx

```diff
@@ -1,13 +1,17 @@
+import { useState } from 'react';
 import SEO from '../components/SEO';
 import Banner from '../components/Banner';
 import Posts from '../components/Posts';
+import NavBar from '../components/NavBar';
 
 const Home = () => {
+  const [bannerIsVisible, setBannerVisibility] = useState<boolean>(true);
```

Comment:
You can drop the `<boolean>` here, since it can infer that it's a boolean based on the initial value of `true`.